### DISCUSSION TITLE
Keeping the user changed values in the presets (json files in the presets folder)

### DISCUSSION THREAD

**Discussion by vaclavmuller** [lllyasviel/Fooocus#3365] [Q&A] [answered]
I like the program preferences, but I changed these values:
    "default_performance": "Quality",
    "default_aspect_ratio": "1344*768", (approximate aspect ratio of my monitor)

Unfortunately, these changes are reverted to the default values ​​during the automatic update.
Would it be possible to set, for example, that these values ​​do not change?

**Comment by mashb1t** [comment, marked as answer]
If you set this in your config.txt they won't get reverted. You may also copy an existing preset and add your customisations to the new file, which is update-proof and doesn't get deleted.
Please always check the terminal logs when encountering any issues as they most likely state the exact error why it wasn't possible to load a specific value.
